client/src/client/frame-runner.js

```diff
@@ -34,6 +35,7 @@ async function initTestFrame(e = {}) {
   // eslint-disable-next-line no-inline-comments
   const { default: chai } = await import(/* webpackChunkName: "chai" */ 'chai');
   const assert = chai.assert;
+  const helpers = curriculumHelpers;
```

Comment:
```suggestion
  const __helpers = curriculumHelpers;
```
This is the inverse of the problem below (for `test-evaluator`).  In this case user defined variables are overwritten by the test code, so if their code relies on them it could fail. 

While this is less likely to be a problem than for the JavaScript challenges, we should name the helpers consistently between the challenges.

Reply:
Agreed.